L3/CustomViews/CheckBoxes/CheckBoxRow.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  CheckBoxRow.swift
+//  L3
+//
+//  Created by Andrii Kuznietsov on 22.12.2023.
+//
+
+import UIKit
+
+class CheckBoxRow: UIView {
+    
+    private enum Const {
+        static let defaultLabelSize: CGFloat = 17
+        static let defaultCheckboxSize: CGFloat = 30
```

Comment:
I would fix naming a bit 

if it is size it should be size

```suggestion
        static let defaultLabelSize: CGSize = .init(width: <#T##CGFloat#>, height: <#T##CGFloat#>)
```